PURL-TYPES.rst

```diff
@@ -115,6 +115,21 @@ cargo
       pkg:cargo/clap@2.33.0
       pkg:cargo/structopt@0.3.11
 
+codeproject
+-----------
+
+``codeproject`` for CodeProject scripts:
+
+- The default repository is ``https://www.codeproject.com``.
```

Comment:
At a minimum, there must be a package manager or an automated way to download and verify packages to qualify for a package-url. An article or a blog post is not a package.

Reply:
Hi @prabhu , there is not such thing in Code Project. However, there are quite a number of Open Source components published in that site many years ago and these packages are not available anywhere else. Is then the PURL specification restricted only to repositories that offer "a package manager or an automated way to download and verify packages"? That would be quite unfortunate. If that is the case, what would be your advice for naming those components?